dali/pipeline/data/tensor_list_test.cc

```diff
@@ -61,6 +61,7 @@ class TensorListTest : public DALITest {
 
   /**
    * Initialize & check a TensorList based on an input shape
+   * Allocate it as float
```

Comment:
Why float?

Reply:
I just added a comment because I was surprised by the behaviour.

Reply:
This is just a thing to consider - maybe we should make other data type a default one?